entrypoint.sh

```diff
@@ -89,7 +93,7 @@ if [[ -n "$JEKYLL_BUILD_PRE_COMMANDS" ]]; then
   echo "Running pre commands..."
   eval "$JEKYLL_BUILD_PRE_COMMANDS"
 fi
-bundle exec jekyll build
+bundle exec jekyll build $JEKYLL_BUILD_OPTS
```

Comment:
You should escape the `$JEKYLL_BUILD_OPTS` portion with quotes or the linter will complain:

```suggestion
bundle exec jekyll build "$JEKYLL_BUILD_OPTS"
```

Reply:
Done in 7615fc5f6d1c0a69bb3f6f7b7bbbc4f312d82b11.